app/components/arclight/search_result_breadcrumbs_component.html.erb

```diff
@@ -1,3 +1,7 @@
-<div class='breadcrumb-links'>
-  <%= rendered_breadcrumbs %>
-</div>
+<dt>
```

Comment:
I like where you're going with it, but as far as I understand it, a `<dd>` can't be a child of `<dt>`.  The missing indentation makes it a bit confusing. I wonder if a small tweak like this would work:

```
<dt class="visually-hidden">Collection Context</dt>
<dd>...</dd>
```

Reply:
@seanaery That seems to work. I've pushed up the change.